spec/rspec/core/option_parser_spec.rb

```diff
@@ -59,6 +59,20 @@ def generate_help_text
       expect { generate_help_text }.to_not output(useless_lines).to_stdout
     end
 
+    describe "-I" do
+      it "sets the path" do
+        options = Parser.parse(%w[-I path/to/foo])
+        expect(options[:libs]).to eq %w[path/to/foo]
+      end
+
+      context "with multiple paths" do
+        it "sets the paths" do
+          options = Parser.parse(%w[-I path/to/foo -I path/to/bar:path/to/baz])
+          expect(options[:libs]).to eq %w[path/to/foo path/to/bar path/to/baz]
+        end
```

Comment:
This spec is failing on windows:

https://ci.appveyor.com/project/rspec/rspec-core/build/325/job/vk19t56q01gmfrpf#L268

It make sense, since windows uses a different `File::PATH_SEPARATOR`.  I think you can get it to pass by changing the first line to:

``` ruby
%W[-I path/to/foo -I path/to/bar#{File::PATH_SEPARATOR}path/to/baz]
```